src/analytics/formTracker/formTracker.ts

```diff
@@ -18,6 +18,10 @@ export class FormTracker extends BaseTracker {
    */
   public getFormFields(form: HTMLFormElement): FormField[] {
```

Comment:
Because you're adding a new scenario in this function, you have also to update the unit tests of this file.

Reply:
I have added unit tests for my getSectionValue function

Reply:
I realised I also have to write a unit test for the checkboxGroup scenario , but in writing this I have discovered an issue i think , see below

Reply:
Sorry I was not clear enough. Because you added a new specific scenario into getFormFields, you need to test (with a new unit test) if when we have a checkbox group, the function (getFormFields)returns what it supposed to return.
Look the test called "getFields should return a list of fields objects" and do the same for a checkbox with two items already checked